RecoLocalCalo/HcalRecAlgos/interface/HcalSimpleRecAlgo.h

```diff
@@ -71,6 +72,7 @@ class HcalSimpleRecAlgo {
   HBHERecHit reconstructHBHEUpgrade(const HcalUpgradeDataFrame& digi,  int first, int toadd, const HcalCoder& coder, const HcalCalibrations& calibs) const;
 
   HFRecHit reconstruct(const HFDataFrame& digi,  int first, int toadd, const HcalCoder& coder, const HcalCalibrations& calibs) const;
+  HFRecHit reconstructQIE10(const QIE10DataFrame& digi,  int first, int toadd, const HcalCoder& coder, const HcalCalibrations& calibs) const;
   HFRecHit reconstructHFUpgrade(const HcalUpgradeDataFrame& digi,  int first, int toadd, const HcalCoder& coder, const HcalCalibrations& calibs) const;
```

Comment:
Many interfaces in the HCAL digi and reco code could be simplified by templating, but I decided that this PR would already have so many changes, it was better not to confound things at the moment.